content/bn/docs/setup/production-environment/tools/kubeadm/_index.md

```diff
@@ -0,0 +1,4 @@
+---
+title: "kubeadm সাহায্যে ক্লাস্টার বুটস্ট্রাপ করা হয়"
```

Comment:
kubeadm এর সাহায্যে ক্লাস্টার বুটস্ট্রাপ করা

Reply:
fixed it